src/components/ContactUs/DesktopVersion.tsx

```diff
@@ -42,17 +72,41 @@ export default function DesktopVersion() {
           </label>
           <textarea
             id="message"
+            name="message"
             placeholder="Hello, I'm getting in touch ..."
             className="border border-[#CEEDF4] w-full rounded-lg px-4 py-[22px] mb-9 h-[178px] resize-none outline-none placeholder-[#7E7979] font-montserrat text-base"
             required
           />
+          <ValidationError
+            prefix="Message"
+            field="message"
+            errors={state.errors}
+          />
           {/*//TODO: replace this button with the app button component */}
```

Comment:
This `TODO` is straying away, we should move the stuff between this TODO and the button so that it stays relevant. I also don't know if it is a small enough change for you to fix it in this PR and then get rid of the TODO. It is fine either way.

The same applies to the mobile version below.

Reply:
@antosan made changes as per the review with the latest commit.